app/index.html

```diff
@@ -10,9 +10,28 @@
     <link href='https://fonts.googleapis.com/css?family=Lato:400,400italic,700,700italic,900,900italic,300italic,300,100italic,100' rel='stylesheet' type='text/css'>
     <style type='text/css' href='app.css'></style>
     <style>
+    label, button {
```

Comment:
```
+    label, button {
 +      display: block;
 +      margin-bottom: 10px;
 +    }
 +
 +    .column {
 +      float: left;
 +      width: 50%;
 +      padding: 10px;
 +
 +      -webkit-box-sizing: border-box;
 +      -moz-box-sizing: border-box;
 +      box-sizing: border-box;
 +    }
 +
 +    #left {
 +      border-right: 1px solid #333;
 +    }
 +
```

Can we put this into app.scss instead of inlining?